suite-native/biometrics/src/useBiometrics.tsx

```diff
@@ -79,7 +77,9 @@ export const useBiometrics = () => {
         if (!isBiometricsAuthenticationCanceled) {
             handleAuthentication();
         }
-    }, [handleAuthentication, isBiometricsAuthenticationCanceled]);
+        // only run on change of isBiometricsAuthenticationCanceled
```

Comment:
Why? Wouldn't that call handleAuthentication with wrong params memoized?

Reply:
I've simplified how things work once more and deleted this part.
https://github.com/trezor/trezor-suite/commit/17d04cf14b6d2df4d0852553892b3736b442e4c6